docs/resources/vault_secrets_integration_azure.md

```diff
@@ -0,0 +1,75 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_vault_secrets_integration_azure Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The Vault Secrets Azure integration resource manages an Azure integration.
+---
+
+# hcp_vault_secrets_integration_azure (Resource)
+
+The Vault Secrets Azure integration resource manages an Azure integration.
+
+## Example Usage
+
+```terraform
+resource "hcp_vault_secrets_integration_azure" "example" {
+  name         = "my-azure-1"
+  capabilities = ["ROTATION"]
+  client_secret = {
+    "tenant_id" : "7eb3...",
+    "client_id" : "9de0...",
+    "client_secret" : "WZk8..."
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `capabilities` (Set of String) Capabilities enabled for the integration. See the Vault Secrets documentation for the list of supported capabilities per provider.
```

Comment:
as dicussed in a preivous PR, i didnt foudn this documentaiton

Reply:
This is a standard message across all integrations. [AWS](https://github.com/hashicorp/terraform-provider-hcp/blob/main/docs/resources/vault_secrets_integration_aws.md#:~:text=hcp_vault_secrets_integration_aws%20(Resource)-,The%20Vault%20Secrets%20AWS%20integration%20resource%20manages%20an%20AWS%20integration.,-Example%20Usage), [GCP](https://github.com/hashicorp/terraform-provider-hcp/blob/main/docs/resources/vault_secrets_integration_gcp.md#:~:text=hcp_vault_secrets_integration_gcp%20(Resource)-,The%20Vault%20Secrets%20GCP%20integration%20resource%20manages%20an%20GCP%20integration.,-Example%20Usage), [Twilio](https://github.com/hashicorp/terraform-provider-hcp/blob/main/docs/resources/vault_secrets_integration_twilio.md#:~:text=hcp_vault_secrets_integration_twilio%20(Resource)-,The%20Vault%20Secrets%20Twilio%20integration%20resource%20manages%20a%20Twilio%20integration.,-Example%20Usage)

Reply:
Do you think linking users to our docs' landing page would be helpful?

https://developer.hashicorp.com/hcp/docs/vault-secrets

Reply:
I could do that, the docs page doesn't directly talk about the capabilities option though. Another option would be to list the options `[ 'sync', 'rotation', 'dynamic']` in the same string. I am fine with adding either of these. 

Note: This is a generated file, hence making this change will affect all the integration doc files where capabilities is listed.